manager/testdata/e2e/module-read.yaml

```diff
@@ -5,11 +5,12 @@ apiVersion: app.m4d.ibm.com/v1alpha1
 kind: M4DModule
 metadata:
   name: arrow-flight-module
+  namespace: default
   labels:
     name: arrow-flight-module
     version: 0.0.1  # semantic version
 spec:
-  chart: ghcr.io/the-mesh-for-data/arrow-flight-module-chart:latest
+  chart: kind-registry:5000/m4d-system/arrow-flight-module-chart:latest
```

Comment:
quick question, why do we have to put it in the local registry?

Reply:
For some reason the manager did not trust ghcr.io ... I did not check in depth why it was signed by an unknown authority but just uploaded it to the local registry that is trusted to make some progress. But you're right we should probably tackle that problem.

Reply:
Ok. I fixed the certificate issue!